packages/plugin-bundle-webpack/plugin.js

```diff
@@ -21,9 +21,22 @@ async function compilePromise(webpackConfig) {
 module.exports = function plugin(config, args) {
   return {
     async bundle({ srcDirectory, destDirectory, log, jsFilePaths }) {
-      let homepage = config.homepage || "";
+      let homepage = config.homepage || "/";
       let fallback = config.devOptions?.fallback || "index.html";
 
+      const jsOutputPattern = args?.outputPatterns?.js || "js/bundle-[hash].js";
```

Comment:
until we add a build process for these, I don't believe Node v10 or v12 support nullish coalescing via `?`, and we need to support down to Node v10

Reply:
Okay, switched the optional chaining out for an alternative.